src/constants/markets.ts

```diff
@@ -39,16 +39,15 @@ export enum MarketFilters {
   ALL = 'all',
   NEW = 'new',
   PREDICTION_MARKET = 'Prediction Market',
-  FX = 'FX',
-  LAYER_1 = 'Layer 1',
-  LAYER_2 = 'Layer 2',
-  DEFI = 'Defi',
-  AI = 'AI',
-  NFT = 'NFT',
-  GAMING = 'Gaming',
-  MEME = 'Meme',
-  RWA = 'RWA',
-  ENT = 'Entertainment',
+  FX = 'fiat',
```

Comment:
[q] is this data coming from abacus? curious as to why we're updating most of these enum k/v pairs

Reply:
This data is coming back from Abacus after it fetches from the metadata service. I re-added the old tags so we don't have any compatibility issues when using `configs/markets.json`

Reply:
dope yeah i was slightly concerned about that. thanks man!